assume/common/utils.py

```diff
@@ -172,6 +173,30 @@ def plot_orderbook(orderbook: Orderbook, results: list[dict]):
     return fig, ax
 
 
+def visualize_orderbook(order_book: Orderbook, hour_count: int = 24):
```

Comment:
What exactly should be displayed here?
When running the tests, no "block_id" or "link" is given, so the line 185 returns an error.